documentation/modules/configuring/con-skipping-scale-down-checks.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// con-scaling-kafka-clusters.adoc
+
+[id='con-skipping-scale-down-checks-{context}']
+= Skipping checks on scale-down operations
+
+[role="_abstract"]
+By default, Strimzi performs a check to ensure that there are no partition replicas on a broker before initiating a scale-down operation on a Kafka cluster. 
+If replicas are found, the scale-down operation is blocked to prevent potential data loss. 
+To fix this, no replicas must be left on the broker before trying to scale it down again.
+
+However, there may be scenarios where you want to bypass this blocking mechanism.
+Disabling the check might be necessary on busy clusters, for example, because new topics keep generating replicas for the broker.
+This situation can indefinitely block the scaling down of clusters, even when brokers are nearly empty. 
+
+You can bypass the blocking mechanism by annotating the `Kafka` resource for the Kafka cluster.
+Annotate the resource by setting the `strimzi.io/skip-broker-scaledown-check` annotation to `true`:
+
+.Adding the annotation to skip checks on scale-down operations  
+[source,shell,subs="+quotes,attributes+"]
+----
+kubectl annotate Kafka my-kafka-cluster strimzi.io/skip-broker-scaledown-check="true"
+----
+
+This annotation instructs Strimzi to skip the scale-down check.
+Replace `my-kafka-cluster` with the name of your specific `Kafka` resource.
+
+To restart the check, remove the annotation:
```

Comment:
```suggestion
To re-enable the check for next scale-downs, remove the annotation:
```

Reply:
thought _restore_ might be a better word